pontoon/machinery/utils.py

```diff
@@ -56,18 +59,37 @@ def get_google_translate_data(text, locale_code):
         }
 
 
-def get_translation_memory_data(text, locale, pk=None):
-    entries = (
-        base.models.TranslationMemoryEntry.objects.filter(locale=locale)
-        .minimum_levenshtein_ratio(text)
-        .exclude(translation__approved=False, translation__fuzzy=False)
+def get_concordance_search_results(text, locale, pk=None):
+    search_list = base.utils.get_search_phrases(text)
+    search_query_list = [(s, locale.db_collation) for s in search_list]
+    search_filters = (
+        Q(target__icontains_collate=(phrase, collation)) | Q(source__icontains=phrase)
+        for phrase, collation in search_query_list
+    )
+    search_filters = (
+        Q(target__icontains_collate=(phrase, collation)) | Q(source__icontains=phrase)
+        for phrase, collation in search_query_list
+    )
```

Comment:
I've seen these 4 lines 4 lines ago. :)

Reply:
Yeah, that's why this PR is marked as a WIP :-) I'm going to clean up all of that.